ClearKeep/Scenes/Register/Views/RegisterContentView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  RegisterContentView.swift
+//  ClearKeep
+//
+//  Created by MinhDev on 04/03/2022.
+//
+
+import SwiftUI
+import CommonUI
+import RealmSwift
+
+private enum Constants {
+	static let width = 120.0
+	static let height = 40.0
+	static let radius = 20.0
+}
+
+struct RegisterContentView: View {
+// MARK: - Variables
+	@Environment(\.colorScheme) var colorScheme
+	@State var username: String
+	@State var password: String
+	@State var displayname: String
+	@State var rePassword: String
+	@State var focused: Bool = false
+	@State var appTheme: AppTheme
+	@State var inputStyle: TextInputStyle
+// MARK: - Body
+	var body: some View {
+		GroupBox(label:
+					Text("Please fill in the information below to complete your sign up")
```

Comment:
@Minhdn1 : Using localized string

```suggestion
					Text("Signup.Title".localize)
```

Reply:
fix complete